routing/road_graph_router.cpp

```diff
@@ -247,10 +249,15 @@ void RoadGraphRouter::ReconstructRoute(vector<Junction> && path, Route & route,
   if (m_directionsEngine)
     m_directionsEngine->Generate(*m_roadGraph, path, times, turnsDir, geometry, cancellable);
 
+  feature::TAltitudes altitudes(path.size());
+  for (size_t i = 0; i < path.size(); ++i)
+    altitudes[i] = path[i].GetAltitude();
+
   route.SetGeometry(geometry.begin(), geometry.end());
   route.SetSectionTimes(times);
   route.SetTurnInstructions(turnsDir);
   route.SetStreetNames(streetNames);
+  route.SetAltitudes(altitudes);
```

Comment:
Wait, wait! You decided to rename methods instead of passing vector by rvalue-reference?